check_actions_status.py

```diff
@@ -0,0 +1,109 @@
+import json
```

Comment:
what is the reason for needing to move the script?

Reply:
The other location wasn’t accessible for running Python. Even with fetch depth the files just weren’t there